src/middleware/navigation/update/has.beneficial.owners.or.managing.officers.update.middleware.ts

```diff
@@ -4,10 +4,12 @@ import { logger } from '../../../utils/logger';
 import { SECURE_UPDATE_FILTER_URL } from '../../../config';
 import { getApplicationData } from "../../../utils/application.data";
 import { checkBOsOrMOsDetailsEnteredUpdate, NavigationErrorMessage } from '../check.condition';
+import { ApplicationData } from 'model';
 
 export const hasBOsOrMOsUpdate = (req: Request, res: Response, next: NextFunction): void => {
   try {
-    if (!checkBOsOrMOsDetailsEnteredUpdate(getApplicationData(req.session))) {
+    const appData: ApplicationData = getApplicationData(req.session);
+    if (!checkBOsOrMOsDetailsEnteredUpdate(appData) && !appData.update?.no_change) {
```

Comment:
Would it be better to check no_change first?

if (!appData.update?.no_change && !checkBOsOrMOsDetailsEnteredUpdate(appData)) {

Reply:
which is 
if (!(appdate.update?.noChange || checkBOsOrMOsDetailsEnteredUpdate(appData))) 
but that's not any clearer?

Reply:
The first option if its for readability alone (and maybe a _tiny_ performance tweak). I think it'll take a short second for anyone reading it no matter which way you write it, its just one of those lines!